pycroft/model/finance.py

```diff
@@ -51,7 +51,7 @@ def __contains__(self, date):
         return date in closed(self.begins_on, self.ends_on)
 
     __table_args__ = (
-        CheckConstraint('begins_on < ends_on'),
+        CheckConstraint('begins_on <= ends_on'),
```

Comment:
I too find this sensible. Just for clarification, do we have a use case for one-point-intervals?